services/web-app/layouts/layout/layout.module.scss

```diff
@@ -27,3 +34,20 @@
   margin: $spacing-08 $spacing-05 $spacing-03;
   @include type-style('label-01');
 }
+
+.header-name {
+  @include breakpoint(lg) {
+    position: absolute;
+  }
+}
+
+.header-navigation {
+  @include breakpoint(lg) {
+    //align nav to grid
+    margin-left: 24.3%;
```

Comment:
I'd approach this differently, so we don't have arbitrary styling values like `24.3%`.

With

```scss
.header-name {
  @include breakpoint(lg) {
    position: absolute;
  }
}

.header-grid {
  @include breakpoint(lg) {
    width: 100%;
    height: 100%;
    margin: 0;
  }
}

.header-navigation {
  margin-left: -$spacing-05;
}
```

We can properly use the `Grid` and `Column` component:

```jsx
            <Header aria-label="Carbon Design System" className={styles.header}>
              <SkipToContent />
              <HeaderMenuButton
                aria-label="Open menu"
                onClick={onClickSideNavExpand}
                isActive={isSideNavExpanded}
              />
              <div className={styles.headerName}>
                <Link href="/assets">
                  <a className="cds--header__name">Carbon Design System</a>
                </Link>
              </div>
              <Grid condensed className={styles.headerGrid}>
                <Column sm={0} lg={{ span: 8, offset: 4 }}>
                  <HeaderNavigation
                    aria-label="Main navigation"
                    className={styles.headerNavigation}
                  >
                    {globalNavData.map((data) => (
                      <HeaderMenuItem
                        key={data.path}
                        isCurrentPage={router.pathname.startsWith(data.path)}
                        href={data.path}
                        element={NextLink}
                      >
                        {data.title}
                      </HeaderMenuItem>
                    ))}
                  </HeaderNavigation>
                </Column>
              </Grid>
            </Header>
```